app/Domain/Integrations/Exceptions/InconsistentIntegrationTypeException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations\Exceptions;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Subscriptions\Subscription;
+use RuntimeException;
+
+final class InconsistentIntegrationTypeException extends RuntimeException
```

Comment:
No need to add the `Exception` suffix. Same reason as why not to add `Interface`, `Abstract`, etc

Reply:
[Remove Exception suffix](https://github.com/cultuurnet/publiq-platform/pull/1362/commits/631ae15bdc135e20c5b986656375d6727f7c75ca)